cmd/telemetrygen/internal/metrics/config.go

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
I think Jaeger Authors should not be added

Reply:
removed.